azure-pipelines.yml

```diff
@@ -23,6 +23,10 @@ jobs:
   pool: Hosted VS2017
 
   steps:
+  - task: DotNetCoreInstaller@0
+    inputs:
+    version: '2.2.105'
```

Comment:
Can we instead define a variable in the global variables and share it on both steps? That way we're always consistent on the version we install for bot jobs.

Reply:
Yes. Will do. Thanks!